cli/scancommands.go

```diff
@@ -377,6 +377,28 @@ func BuildScan(c *components.Context) error {
 	return commandsCommon.Exec(buildScanCmd)
 }
 
+func getSubScansToPreform(c *components.Context) (subScans []utils.SubScanType, err error) {
```

Comment:
Maybe this function should be in a utils file since we use it both in scancommands and in gitcommands?

Reply:
Created `utils.go` file and transferred this and some more shared methods